features/tracker-allowlist.json

```diff
@@ -992,6 +992,15 @@
                             "triblive.com - https://github.com/duckduckgo/privacy-configuration/issues/1730"
                         ]
                     },
+                    {
+                        "rule": "doubleclick.net/pixel",
+                        "domains": [
+                          "www.sbs.com.au"
```

Comment:
Worth a quick check of reference tests to see if we support hostname here, but using the domain `sbs.com.au` would be safe either way

Reply:
Yeah, good call, I just dropped "www."